docs/token_transferring/neon_transfer.mdx

```diff
@@ -0,0 +1,373 @@
+  address: '',
+  decimals: NEON_TOKEN_MINT_DECIMALS,
+  name: 'Neon',
+  symbol: 'NEON',
+  logoURI: 'https://raw.githubusercontent.com/neonlabsorg/token-list/main/neon_token_md.png'
+};
+
+const transaction = await neonNeonTransactionEthers({
+  provider,
+  from: neonWallet.address,
+  to: NEON_TRANSFER_CONTRACT_DEVNET,
+  solanaWallet,
+  amount
+});
+
+const signedTrx = await neonWallet.signTransaction(transaction);
+
+if (signedTrx?.rawTransaction) {
+  const txResult = neonWallet.sendSignedTransaction(signedTrx.rawTransaction);
+  txResult.on('transactionHash', (hash: string) => console.log(hash));
+  txResult.on('error', (error: Error) => console.error(error));
+}
+```
+
+</TabItem>
+</Tabs>
+
+> Note: You can find more examples in our [repository](https://github.com/neonlabsorg/neon-client-transfer) in the `examples` folder.
+
+### 2.2 Transfer SOL
+
+To transfer SOL as a native token, it is necessary to change the configuration of Neon EVM.
+
+```javascript
+// Neon EVM Devnet for SOL fee configuration
+const proxyUrl = `https://devnet.neonevm.org/sol`;
+const solProxyRpcApi = new NeonProxyRpcApi(proxyUrl);
+const solProxyStatus = await solProxyRpcApi.evmParams();
+const gasTokens = await solProxyRpcApi.nativeTokenList();
+
+const id = gasTokens.findIndex(i => i.token_name === 'SOL');
+const gasToken = gasTokens[id];
+
+const neonEvmProgram = new PublicKey(solProxyStatus.NEON_EVM_ID);
+const solTokenMint = new PublicKey(gasToken.token_mint);
+const chainId = parseInt(gasToken.token_chain_id, 16);
+```
+
+<Tabs>
+<TabItem value="Opt1" label="Solana to Neon EVM" default>
+
+To transfer SOL token as a native token from Solana to Neon EVM, use the following patterns:
+
+```javascript
+import { solanaSOLTransferTransaction, SPLToken } from '@neonevm/token-transfer-core';
+
+const neonToken: SPLToken = {
+  chainId: solChainId,
+  address_spl: solTokenMint.toBase58(),
+  address: '0xc7Fc9b46e479c5Cb42f6C458D1881e55E6B7986c',
+  decimals: 9,
+  name: 'Solana SOL',
+  symbol: 'SOL',
+  logoURI: 'https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/solana-sol-logo.svg'
+};
+
+const amount = '1';
+
+const transaction = await solanaSOLTransferTransaction({
+  connection,
+  solanaWallet: solanaWallet.publicKey,
+  neonWallet: neonWallet.address,
+  neonEvmProgram: solEvmProgram,
+  neonTokenMint: solTokenMint,
+  splToken: solToken,
+  amount,
+  chainId: solChainId
+});
+
+const signers = [toSigner(solanaWallet)];
+transaction.recentBlockhash = (await connection.getLatestBlockhash('finalized')).blockhash; // Network blockhash
+transaction.sign(...signers);
+const signature = await connection.sendRawTransaction(transaction.serialize());
+console.log(signature);
+```
+
+</TabItem>
+<TabItem value="Opt2" label="Neon EVM to Solana" default>
+
+Similarly, to transfer SOL tokens from Neon EVM to Solana:
+
+```javascript
+import {
+  SOL_TRANSFER_CONTRACT_DEVNET,
+} from '@neonevm/token-transfer-core';
+
+const neonToken: SPLToken = {
```

Comment:
Here we don't need to have solToken, as. it is native, we don't pass it in the transaction. You can remove it. Remove all this:
```
const neonToken: SPLToken = {
  chainId: solChainId,
  address_spl: solTokenMint.toBase58(),
  address: '0xc7Fc9b46e479c5Cb42f6C458D1881e55E6B7986c',
  decimals: 9,
  name: 'Solana SOL',
  symbol: 'SOL',
  logoURI: 'https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/solana-sol-logo.svg'
};
```